### PR TITLE
Add unittest2pytest recipe

### PR DESCRIPTION
**Pull request by nicoddemus**
@htgoebel, @gqmelo, would you like to be added as maintainer of the recipe as well?

### CODE REVIEW

**Comment by gqmelo** [comment]
@nicoddemus, you can add me

File: recipes/unittest2pytest/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  run:
+    - python
+
+test:  
+  commands:
+    - unittest2pytest --help
+
+about:
+  home: https://github.com/pytest-dev/unittest2pytest
+  license: GPL
```
Comment (scopatz) [inline recipes/unittest2pytest/meta.yaml]: Can you please include the `license_file` too

Comment (nicoddemus) [inline recipes/unittest2pytest/meta.yaml]: Done, thanks for noticing.

File: recipes/unittest2pytest/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
```
Comment (nicoddemus) [inline recipes/unittest2pytest/meta.yaml]: Oh thanks, I didn't know that. Updated.

**Comment by nicoddemus** [comment]
Thanks!

**Comment by htgoebel** [comment]
> would you like to be added as maintainer of the recipe as well?

No thanks.

**Pull request merged**